### PR TITLE
Problem: missing support NFT(#14)

### PR DESCRIPTION
**Pull request by linfeng-crypto**

### CODE REVIEW

**Comment by tomtau** [comment]
should it also update the changelog?

**Comment by linfeng-crypto** [comment]
> should it also update the changelog?

update the changelog when publishing a new version?

File: chainlibpy/transaction.py [hunk 1]
```diff
@@ -67,6 +67,7 @@ def _sign(self) -> str:
             sign_doc.to_dict(), separators=(",", ":"), sort_keys=True
         )
         message_bytes = message_str.encode("utf-8")
+        print(list(message_bytes))
```
Comment (tomtau) [inline chainlibpy/transaction.py]: is this needed? or can use some debug log level, so one can turn it off?

**Review by tomtau — CHANGES_REQUESTED** [review]
> update the changelog when publishing a new version?
not sure of the release process here -- would that be in a separate PR?

https://github.com/crypto-org-chain/chainlibpy/pull/16#issuecomment-888915697

**Pull request merged**